CryptoTechReminderSystem.Test/RemindLateDevelopersTests.cs

```diff
@@ -152,7 +105,11 @@ public void CanRemindDevelopersDirectlyWithAMessage()
         {
             var remindDeveloperSpy = new RemindDeveloperSpy();
             var getLateDevelopersStub = new GetLateDevelopersStub();
-            var clock = new Clock();
+            var clock = new ClockStub(
+                new DateTimeOffset(
+                    new DateTime(2019, 03, 01, 10, 30, 0)
+                )
+            );
```

Comment:
This can be extracted to setup

Reply:
Fixed in https://github.com/chubberlisk/crypto_tech/pull/17/commits/3dcb83149182595a46d6fbe01872e1d4ae9cef67